app/controllers/submit.js

```diff
@@ -279,6 +281,27 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
         const modelDOI = this.get('model.doi');
         return (basicsDOI || modelDOI) && basicsDOI !== modelDOI;
     }),
+    basicsLicense: Ember.computed('model', function() {
+        let record = this.get('model.licenseRecord');
+        let license = this.get('model.license');
+        return {
+            year: record ? record.year : null,
+            copyrightHolders: record && record.copyright_holders ? record.copyright_holders.join(',') : null,
+            licenseType: license || null
+        }
+    }),
+    licenseChanged: Ember.computed('model.license', 'model.licenseRecord', 'model.license', function() {
```

Comment:
should have basicsLicense as a property.  licenseChanged will need to compare the current license on the preprint (that is saved on the server), to whatever the pending license information is.  So model.license to 'basicsLicense'.